app/controllers/MathSwipeController.coffee

```diff
@@ -0,0 +1,58 @@
+InputSolver         = require '../services/InputSolver'
+GameGrid            = require '../models/GameGrid'
+ExpressionGenerator = require '../services/ExpressionGenerator'
+Board               = require '../views/Board'
+Tuple               = require '../models/Tuple'
+$                   = require 'jQuery'
```

Comment:
I get a warning because this ```jQuery``` is capitalized and the one that's required is just ```jquery```